src/components/Person/Person.jsx

```diff
@@ -1 +1,53 @@
-// export const Person = ({ person }) => ();
+import React from 'react';
+import PropTypes from 'prop-types';
+
+export const Person = ({ person }) => {
+  const {
+    name,
+    age,
+    sex,
+    isMarried,
+    partnerName,
+  } = person;
+  const partnerStatus = (sex === 'm')
+    ? 'wife'
+    : 'husband';
+
+  return (
+    <section className="Person">
+      <h2 className="Person__name">{`My name is ${name}`}</h2>
+
+      {age && (
+        <p className="Person__age">{`I am ${age}`}</p>
+      )}
+
+      <p className="Person__partner">
+        {
+          isMarried
+            ? (`${partnerName} is my ${partnerStatus}`)
+            : (`I am not married`)
+        }
+      </p>
+    </section>
+  );
+};
+
+Person.propTypes = {
+  person: PropTypes.shape({
+    name: PropTypes.string,
+    age: PropTypes.number,
+    sex: PropTypes.string,
+    isMarried: PropTypes.bool,
+    partnerName: PropTypes.string,
+  }),
+};
+
+Person.defaultProps = {
+  person: {
+    name: 'Jeanne Louise Calment',
+    age: 122,
+    sex: 'f',
+    isMarried: true,
```

Comment:
I'm not sure, will it works correct, cause `age` and `isMarried` can be omitted , but you set default value for it

Reply:
🤔 thanks, it is interesting
i checked this case
the default value will only be replaced by defaultProps if no object is passed. If an object is passed without a property, the property will be undefined.
It is possible to set a default value for a property during destructuring. I think this is not necessary in this task, but this is not accurate)